snuba/api.py

```diff
@@ -40,7 +41,14 @@ def check_down_file_exists():
 
 def check_clickhouse():
     try:
-        return any(settings.CLICKHOUSE_TABLE == r[0] for r in clickhouse_ro.execute('show tables'))
+        clickhouse_tables = clickhouse_ro.execute('show tables')
+        for name in get_enabled_dataset_names():
+            dataset = get_dataset(name)
+            dist_table = dataset.SCHEMA.get_table_name()
```

Comment:
`dist_table` should probably just be `table_name` here on L47-48 (since this is not necessarily actually the distributed table, depending on the mode.)

Reply:
fixed